tools/autograd/derivatives.yaml

```diff
@@ -249,6 +250,7 @@
 
 - name: as_strided(Tensor(a) self, int[] size, int[] stride, int? storage_offset=None) -> Tensor(a)
   self: as_strided_backward(grad, TensorGeometry(self), size, stride, storage_offset)
+  result: linear
```

Comment:
Add a test for `element_wise` / `linear`?

Reply:
You mean to make sure the gradients are correct?
I prefer to delay that until there is gradcheck higher in the stack (I'll merge the whole thing as at once. So this won't be in master without being tested!)